src/test/java/teammates/test/pageobjects/InstructorCourseEditPage.java

```diff
@@ -232,8 +232,10 @@ public boolean clickShowNewInstructorFormButton() {
         return isFormShownCorrectly;
     }
     
-    public boolean clickOnAccessLevelViewDetails(String role) {
-        WebElement viewDetailsLink = browser.driver.findElement(By.cssSelector(
+    public boolean clickOnAccessLevelViewDetails(String formId, String role) {
```

Comment:
Then again this file is in a huge state of disarray (you know something is fishy when you `assertTrue` on a method that clicks something), maybe create an issue _specifying_ what refactoring has to be done.